app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if command_list[0] == "cp":
```

Comment:
You should add a check to ensure that `command_list` contains exactly three elements and that the first element is 'cp'. This is necessary to validate the input as per the checklist requirements.